Dockerfile

```diff
@@ -1,5 +1,6 @@
 FROM python:3.5
 RUN mkdir /var/run/sshd
 RUN chmod 0755 /var/run/sshd
+RUN apt-get install vim
```

Comment:
I would just leave this out. If you do want to add more deps like this, try to do it all in the same line (i.e the one below)